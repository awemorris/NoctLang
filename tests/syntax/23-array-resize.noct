func main() {
    var arr1 = [];
    arr1[63] = 123;

    print(arr1[1]);
    print(arr1[63]);

    var arr2 = [];
    arr2->resize(32);
    print(arr2[1]);  
    print("size = " + arr2.length);

    var arr3 = newArray(64);
    print("size = " + arr3.length);
}
