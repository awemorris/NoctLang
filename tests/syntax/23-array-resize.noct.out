0
123
0
size = 32
size = 64
